posthog/hogql_queries/experiments/experiment_trends_query_runner.py

```diff
@@ -89,12 +95,18 @@ def _get_insight_date_range(self) -> InsightDateRange:
             explicitDate=True,
         )
 
-    def _get_breakdown_filter(self) -> BreakdownFilter:
+    def _get_event_breakdown_filter(self) -> BreakdownFilter:
         return BreakdownFilter(
             breakdown=self.breakdown_key,
             breakdown_type="event",
         )
 
+    def _get_data_warehouse_breakdown_filter(self, column_name: str) -> BreakdownFilter:
+        return BreakdownFilter(
+            breakdown=f"{column_name}.properties.`{self.breakdown_key}`",
```

Comment:
@jurajmajerik As it turns out, this syntax isn't supported yet. See some conversation here: https://posthog.slack.com/archives/C019RAX2XBN/p1732050126051829